test/Chainweb/Test/Pact/RemotePactTest.hs

```diff
@@ -82,45 +89,76 @@ tests = do
     newPeerInfo <- readMVar peerInfoVar
     let thePort = _hostAddressPort (_peerAddr newPeerInfo)
 
+    let cmds = apiCmds cwVersion chainId0
+    let cwBaseUrl = getCwBaseUrl thePort
+    cwEnv <- getClientEnv cwBaseUrl
+    (tt0, rks) <- testSend cmds cwEnv
+
+    tt1 <- testPoll cmds cwEnv rks
+
+    let valCmd = MempoolLookupApiCmd (lookupClient cwVersion chainId0)
```

Comment:
You could also use `Chainweb.Mempool.RestAPI.Client.toMempool` and use the remote mempool functions directly

Reply:
Ok, i'll look at that again.  I saw that but wasn't sure what is was for.